modules/raft/src/main/java/org/apache/ignite/internal/raft/server/impl/JraftServerImpl.java

```diff
@@ -121,8 +134,12 @@ public JraftServerImpl(ClusterService service, Path dataPath, NodeOptions opts)
         this.opts.setSharedPools(true);
 
         if (opts.getServerName() == null) {
-            opts.setServerName(service.localConfiguration().getName());
+            this.opts.setServerName(service.localConfiguration().getName());
         }
+
+        this.opts.setElectionTimeoutStrategy(
+                new ExponentialBackoffTimeoutStrategy(this.opts.getElectionTimeoutMs(), ELECTION_TIMEOUT_MS_MAX,
```

Comment:
Use the default constructor:
new ExponentialBackoffTimeoutStrategy(this.opts.getElectionTimeoutMs())